sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSink.scala

```diff
@@ -128,6 +128,7 @@ class FileStreamSink(
         hadoopConf = hadoopConf,
         partitionColumns = partitionColumns,
         bucketSpec = None,
+        statsTrackers = Nil,
```

Comment:
I think `FileStreamSink` is not a `RunnableCommand`? #18159 adds data writing metrics for certain `RunnableCommand`.

Do we have a physical node representing data writing operation in a `FileStreamSink` so we can bind its `SQLMetric` and update the metrics after insertion? Looks like the `addBatch` accepts arbitrary `DataFrame` and writes the data from it directly.

I'd think it might be another issue.